src/aosm/azext_aosm/cli_handlers/onboarding_cnf_handler.py

```diff
@@ -69,23 +70,24 @@ def output_folder_file_name(self) -> str:
         """Get the output folder file name."""
         return CNF_OUTPUT_FOLDER_FILENAME
 
-    def _get_input_config(self, input_config: dict = None) -> OnboardingCNFInputConfig:
+    def _get_input_config(self, input_config: Optional[dict] = None) -> OnboardingCNFInputConfig:
         """Get the configuration for the command."""
         if input_config is None:
             input_config = {}
         return OnboardingCNFInputConfig(**input_config)
 
-    def _get_params_config(self, config_file: Path = None) -> CNFCommonParametersConfig:
+    def _get_params_config(self, config_file: Path) -> CNFCommonParametersConfig:

```

Comment:
not 100% sure this is an ok change, will look and delete this if fine

Reply:
It has to be, because you can't call the first line of the method (`with open(config_file, "r", encoding="utf-8") as _file:`) if `Path` is `None`.

If you think there really is an issue here, we should discuss, as it suggests whatever might be calling this method actually shouldn't.